src/data/finsupp/to_dfinsupp.lean

```diff
@@ -187,4 +187,68 @@ noncomputable def finsupp_lequiv_dfinsupp
   map_add' := finsupp.to_dfinsupp_add,
   .. finsupp_equiv_dfinsupp}
 
+section sigma
```

Comment:
```suggestion

/-- ### Stronger versions of `finsupp.split` -/
```
It would be good to mention these in the module docstring too
section sigma
```